api-report/container-runtime.api.md

```diff
@@ -356,7 +356,9 @@ export interface IGCStats {
 // @public
 export interface IGeneratedSummaryStats extends ISummaryStats {
     readonly dataStoreCount: number;
+    readonly gcBlobNodeCount?: number;
     readonly gcStateUpdatedDataStoreCount?: number;
+    readonly gcTotalBlobSize?: number;
```

Comment:
call this `gcTotalBlobsSize` with blobs in plural as there can be multiple blobs.